spec/requests/contact_embed_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+RSpec.describe "Contact Embed" do
+  let(:edition) do
+    body_field = build(:field, id: "body", type: "govspeak")
+    document_type = build(:document_type, contents: [body_field])
+    create(:edition, document_type_id: document_type.id)
+  end
+
+  describe "GET /documents/:document/contact-embed" do
+    it "renders a service unavailable response when the Publishing API is down" do
```

Comment:
I might be missing something, but I think there's a slight risk here that we're no longer checking the error behaviour through the modal: although the responses will be correct, I'm not sure if we check the modal interprets it correctly, since it's not something that will "just work".

Reply:
I think the same may be true with requirements issues ([below](https://github.com/alphagov/content-publisher/pull/1626/commits/08fbaa896852f386064b2c1446aa5f207c6faefc#diff-eef72a8e83aead427b576a42b7b5b8b8R63)).

Reply:
This should now be resolved by the modal-tests PR: https://github.com/alphagov/content-publisher/pull/1641